docs/modules/operation/pages/geographical-map/introduction.adoc

```diff
@@ -0,0 +1,36 @@
+OpenNMS includes a geographical map which can show nodes with address asset information on a world map, including dynamic alarm status.
```

Comment:
This section doesn't have a header so it is not showing up in the table of contents. 
Suggest
= Geographical Maps